recipe/meta.yaml

```diff
@@ -44,7 +44,7 @@ requirements:
     # To have the thrift compiler as a build tool.
     - thrift-cpp
   host:
-    - abseil-cpp 20230802.0  # [not osx]
+    - abseil-cpp 20211102.0  # [not osx]
```

Comment:
Was this required to fix the conflict?

Reply:
Exactly. With a newer version `20230802.0`, there will be a dependency conflict. Our current recipe of v11.0.0 has `abseil-cpp 20211102.0  # [not osx]` https://github.com/AnacondaRecipes/arrow-cpp-feedstock/blob/38626904e47c5024e687004ce40e08d35448a991/recipe/meta.yaml#L47

Reply:
yes, it has because we haven't done yet that epic https://anaconda.atlassian.net/browse/PKG-2637 ?

Reply:
I caused a mess mistakenly pushing commits to the master-10.0.1 branch before opening the PR

Reply:
Ok, I'm fine with this if we need to do this because of a conflict. Can you add a comment to make sure it's clear that this version is used to fix a conflict?

Reply:
I've added a comment to clarify the usage of abseil-cpp 20211102.0